src/lib.rs

```diff
@@ -1,5 +1,55 @@
 //! Winit is a cross-platform window creation and event loop management library.
 //!
+//! # Installation
```

Comment:
we're not a binary, so this section looks odd.

Reply:
Changed to "usage"